src/traced/probes/ftrace/cpu_reader_fuzzer.cc

```diff
@@ -50,17 +50,20 @@ void FuzzCpuReaderProcessPagesForDataSource(const uint8_t* data, size_t size) {
   memcpy(g_page, data, std::min(size_t(base::GetSysPageSize()), size));
 
   FtraceMetadata metadata{};
```

Comment:
There seems a lot of weird diffs here because of code formatting. Can we do a format pass first?